hx_lti_initializer/static/vendors/development/video.dev.js

```diff
@@ -6309,6 +6320,10 @@ vjs.TextTrackMenuItem = vjs.MenuItem.extend({
 vjs.TextTrackMenuItem.prototype.onClick = function(){
   vjs.MenuItem.prototype.onClick.call(this);
   this.player_.showTextTrack(this.track.id_, this.track.kind());
+  if (typeof(jQuery.publish) === "function") {
```

Comment:
Also, we've been looking at [Able Player](https://ableplayer.github.io/ableplayer/) as of late. I might play around with that a bit before committing to this since I might be starting from scratch regardless. We've been trying to find something that's a bit more centered around accessibility as of late and came across it.